bin/pull-image

```diff
@@ -4,12 +4,19 @@
 
 source bin/lib.sh
 
-if [[ -z "${USE_LOCAL_CIVIFORM}" ]]; then
-  echo "Making sure we're up to date with the latest dev... " \
-    "set environment variable USE_LOCAL_CIVIFORM=1 to skip"
+PULL_SOURCE=civiform/civiform-dev:latest
+LOCAL_TAG=civiform-dev
+
+echo "Making sure we're up to date with the latest dev"
 
-  docker pull -q civiform/civiform-dev:latest
-  docker tag civiform/civiform-dev:latest civiform-dev
-else
-  echo "Using local civiform docker images"
+if [[ -z "${USE_LOCAL_CIVIFORM}" ]]; then
+  echo "Set environment variable USE_LOCAL_CIVIFORM=1 to skip"
+  if [[ -z "${PULL_PROGRESS}" ]]; then
+    echo "Set environment variable PULL_PROGRESS=1 to show download progress"
```

Comment:
It's slightly confusing that the instructions to show download progress are only show when they are already being used.  Perhaps make showing progress the default, and you can use a flag to skip.

Reply:
I remember at one point they changed it to not show pull progress, and then I undid that in my local changes and @bion had asked me to revert that

Reply:
Ah, gotcha. could just leave a comment about the flag in the docstring above, then